source/Nevermore/UnaryWhereParameter.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections.Generic;
```

Comment:
Filename doesn't reflect that there are multiple classes here. Split out each class into a file (preferred) or rename the file.

Reply:
Done